.gitignore

```diff
@@ -20,3 +20,4 @@ ror-dependencies-*.zip
 dependencies-*.zip
 _build
 libangelscript_addons*.a
+source/main/gui/AboutWindow.cpp
```

Comment:
Oh, didn't see this one. It's one of the bad things that happened when i updated my local with the  upstream. Will fix asap!